docs/pages/enterprise/fedramp.mdx

```diff
@@ -1,7 +1,7 @@
 ---
-title: FedRAMP compliance for SSH, Kubernetes and Applications
-description: How to configure SSH, Kubernetes and Application access to be FedRAMP compliant, including support for FIPS 140-2
-h1: FedRAMP for SSH and Kubernetes and Application Access
+title: FedRAMP compliance for SSH, Kubernetes, Databases, Desktops and Web Apps.
```

Comment:
I would remove desktops for now as we haven't done any work to verify that desktop access can be FedRAMP compliant.

Reply:
fixed